classes/class-wc-connect-shipping-label.php

```diff
@@ -110,6 +110,23 @@ protected function get_packages( WC_Order $order ) {
 			return $formatted_packages;
 		}
 
+		protected function get_all_packages() {
+			$packages = $this->settings_store->get_packages();
+
+			if ( ! $packages ) {
+				return new stdClass();
```

Comment:
I wonder if we should just start using stdClass more consistently in our PHP code to avoid this issue altogether?